behaviours/dive_control/dive_control/ConvenienceView.py

```diff
@@ -91,6 +93,37 @@ def _update_waypoint(self) -> None:
 
         self._waypoint_pub.publish(self._waypoint_msg)
 
+    def _print_state(self) -> None:
```

Comment:
Instead of calling loginfo for each line, i suggest you collect your lines in a string and use loginfo once at the end. That way 1) less likely that another log will interrupt your print 2) fewer system calls 3) more screen space because only one time/node name will be printed ;)

ex: 
```
s = "Action Server State\n"

s += "No state msg yet.\n"
s += "States:\n"
...
self._loginfo(s)
```